core/src/com/unciv/ui/pickerscreens/TechButton.kt

```diff
@@ -69,8 +69,6 @@ class TechButton(techName:String, private val techManager: TechManager, isWorldS
                     .apply { color = Color.BLACK }.surroundWithCircle(techIconSize))
 
         if (isWorldScreen) rightSide.add(techEnabledIcons)
-        else rightSide.add(techEnabledIcons)
-                .width(techEnabledIcons.children.size * (techIconSize+6f))
-                .minWidth(150f)
+        else rightSide.add(techEnabledIcons).width(225f)
```

Comment:
if you want that Russian language shows very well, please make the width number larger, but I'm not sure it's good or bad.